README.md

```diff
@@ -1,2 +1,88 @@
-# MyProject
-Web application - a platform for creating an online store
+MyProject
```

Comment:
Use markdown features to beautify readme file

Reply:
ok